### ISSUE TITLE
Drifter requires ansible 2.1.1, host has 2.1.1.0

### ISSUE THREAD

**Issue by schtibe** [liip/drifter#201]
Recently I have following error with new drifter installations:

```
Vagrant has automatically selected the compatibility mode '2.0'
according to the Ansible version installed (2.1.1.0).

Alternatively, the compatibility mode can be specified in your Vagrantfile:
https://www.vagrantup.com/docs/provisioning/ansible_common.html#compatibility_mode

The requested Ansible version (2.1.1) was not found on the guest.
Please check the Ansible installation on your Vagrant guest system (currently: 2.1.1.0),
or adapt the provisioner `version` option in your Vagrantfile.
See https://docs.vagrantup.com/v2/provisioning/ansible_common.html#version
for more information.
```

I can avoid that by changing the variable `ansible_version` to  "2.1.1.0" in `virtualization/drifter/Vagrantfile`. (I wonder it has to be that specific, aren't "2.1.1" and "2.1.1.0" kind of the same?)

Anyway, is anybody else having this problem? Could we change it to "2.1.1.0"? Or are there other solutions to this?

**Comment by gildegoma** [comment]
This will be resolved in the [next release](https://github.com/liip/drifter/blob/master/CHANGELOG.md#fixed). See also #198

**Comment by sephii** [comment]
Fixed in 1.5.0.
